### DISCUSSION TITLE
不知道为什么以.xml结尾的源都有问题

### DISCUSSION THREAD

**Discussion by zxc110** [ReadYouApp/ReadYou#4] [Q&A] [answered]
1、可能无法订阅
2、可以订阅但是只有标题

**Comment by Ashinch** [comment, marked as answer]
RSS 的协议很多种，使用的上游解析插件有些协议还不支持，不过刚刚看到更新了，所以下版本我跟进一下。

想了解目前支持的协议，可以看看上游解析插件的仓库说明：[ParseRSS](https://github.com/muhrifqii/ParseRSS)
